python/dgl/graphbolt/utils/sample_utils.py

```diff
@@ -74,6 +75,34 @@ def add_reverse_edges(
         return combined_edges
 
 
+def exclude_seed_edges(
```

Comment:
```suggestion
def exclude_edges(
```

Reply:
As this function always remove seed edges, so exclude_seed_edges is better?

Reply:
SG.